README.md

```diff
@@ -1,2 +1,84 @@
-# ruby-tic-tac-toe
-This is the ruby implementation of the game tic tac toe
+#  Tic Tac Toe OOP build with Ruby (Object Oriented Programming)
+* Original project specification at [The Odin Project] (https://www.theodinproject/lesseons/oop)
+* The purpose of the project is to master object oriented programming with ruby. 
+
+*![image](./tic-tac.gif)
+
+# This project has four milestones
+- Project setup
+- User interface
+- Game logic
+- Instruction for players
+
+
+# Game Rule:
+Tic Tac Toe is Cool Math Games:
+Xs and Os is a paper-and-pencil game for two players, X, O:
+* Players take turns filing the spaces in 3X3 grid.
+* The player who succeeds in placing three of their marks in the horizontal, vertical, or diagonal or wins the game.
+
+the following example game is won by the first player, X:
+![example](https://upload.wikimedia.org/wikipedia/commons/thumb/1/1b/Tic-tac-toe-game-1.svg/1280px-Tic-tac-toe-game-1.svg.png)
+
+## Instructions
+
+**Run `bundle` within this lab's directory before getting started.**
+
+### Project Structure
+
+```bash 
+├── Gemfile
+├── Gemfile.lock
+├── README.md
+├── bin
+│   └── main.rb
+├── lib
+│   └── player.rb
+└── spec
+    ├── 01_tic_tac_toe_spec.rb
+    ├── 02_play_spec.rb
+    ├── 03_cli_spec.rb
+    └── spec_helper.rb
+```
+
+## How to Run the Game
+
+- clone the repository from here : (https://github.com/sinework/ruby-tic-tac-toe.git)
+
+- Open your git bash and clone the repo.
+
+- go to the diroctory of the file and run ruby "main.rb" or "main"
+
+- Good job you are now playing Tic-Tac-Toe game. 👌🙌
```

Comment:
It's preferable to use commands
1. `git clone <repository>` put the repository url here
2. `cd <repository>/bin`
3. `ruby bin/main.rb`

Update your README